man/man7/zfsprops.7

```diff
@@ -1197,6 +1197,18 @@ are equivalent to the
 and
 .Sy noexec
 mount options.
+.It Sy volthreading Ns = Ns Sy on Ns | Ns Sy off
+Controls internal zvol threading.
+The value
+.Sy off
+disables zvol threading, and zvol relies on application threads.
+The default value is
+.Sy on ,
+which enables threading within a zvol.
+Please note that this property will be overridden by
+.Sy zvol_request_sync
+module parameter.
+This property is only applicable to Linux.
```

Comment:
Might want to mention that `volthreading` is only read at zvol creation/import time.  That is, if you set `volthreading` after the volume is available, it will not have an immediate effect.

Reply:
Thank you for reviewing this. `zvol_set_volthreading()` should make the volthreading property take immediate effect.

Reply:
Ah I must have missed that, thanks